src/components/settings/SettingsLMSTab/LMSConfigs/DegreedConfig.jsx

```diff
@@ -13,6 +13,7 @@ const DegreedConfig = ({ id, onClick }) => {
   const [secret, setSecret] = React.useState('');
   const [degreedCompanyId, setDegreedCompanyId] = React.useState('');
   const [degreedBaseUrl, setDegreedBaseUrl] = React.useState('');
+  const [urlValid, setUrlValid] = React.useState(true);
```

Comment:
do we want to default to false, assuming initially url field's value will be empty?

Reply:
No, because further on with draft functionality we will allow blank fields, also if its set to false before the user even clicks on it, it will be red so it seemed aggressive

Reply:
I see, ok so we start clean and I did forget about the draft ability, so yeah this is correct!